modules/custom/az_dam/config/quickstart/editor.editor.basic_html.yml

```diff
@@ -0,0 +1,85 @@
+uuid: 3edeafcb-89a2-4215-b036-7d2f30157afe
+langcode: en
+status: true
+dependencies:
+  config:
+    - filter.format.basic_html
+  module:
+    - ckeditor
+_core:
+  default_config_hash: AqlPmO16LvJI4D0Ih6u4GFQIzqr5OnLgAUSjcUGWk2g
```

Comment:
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.